src/Compilers/Core/Portable/Emit/Context.cs

```diff
@@ -19,5 +20,27 @@ public EmitContext(CommonPEModuleBuilder module, SyntaxNode syntaxNodeOpt, Diagn
             SyntaxNodeOpt = syntaxNodeOpt;
             Diagnostics = diagnostics;
         }
+
+        public bool Filter(IMethodDefinition method)
```

Comment:
I think that would be better. Also consider using more descriptive name ShouldInclude/ShouldExclude, or something similar.